server/graphql/v1/TransactionInterface.js

```diff
@@ -277,40 +276,12 @@ export const TransactionExpenseType = new GraphQLObjectType({
           return transaction.description || expense;
         },
       },
-      privateMessage: {
```

Comment:
At this stage, these properties should be deprecated but not deleted. Just add a `deprecationReason` like in other places in the schema.